content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -47,6 +61,23 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 			return FileDataElementWrapper.getFromElement(this._domElement);
 		},
 
+		/**
+		 * Initializes object fields.
+		 */
+		_init : function() {
+
+			// Retrive file name
+			var fileInfoDom = this._domElement.getElementsByClassName(
+				FileElementWrapper.CLASS_FILE_INFO);
+
+			var fileTextSpan = fileInfoDom[0].getElementsByClassName(
```

Comment:
Because that should not ever happen. For now I would leave it like this, since if fileInfoDom is an empty array, this piece of code will throw an error and inform us that something has been changed on GitHub page.